qa-tests-backend/src/test/groovy/NetworkBaselineTest.groovy

```diff
@@ -277,57 +288,61 @@ class NetworkBaselineTest extends BaseSpecification {
             [])
     }
 
-    @Unroll
-    // TODO: ROX-11126
-    @Ignore
     @Category(NetworkBaseline)
     def "Verify user get for non-existent baseline"() {
         when:
-        "Create initial set of deployments, wait for baseline to populate"
+        "Create initial set of deployments, wait for them to deploy"
         def beforeDeploymentCreate = System.currentTimeSeconds()
-        batchCreate([USER_DEP, BASELINED_USER_CLIENT_DEP])
+        batchCreate([USER_DEP])
         def justAfterDeploymentCreate = System.currentTimeSeconds()
 
         def serverDeploymentID = USER_DEP.deploymentUid
         assert serverDeploymentID != null
 
-        def baselinedClientDeploymentID = BASELINED_USER_CLIENT_DEP.deploymentUid
-        assert baselinedClientDeploymentID != null
-
-        log.info "Deployment IDs Server: ${serverDeploymentID}, " +
-                    "Baselined client: ${baselinedClientDeploymentID}"
+        log.info "Deployment IDs Server: ${serverDeploymentID}"
 
+        // Get the server baseline to simulate a user asking for a baseline prior to
+        // observation ending.
         def serverBaseline = NetworkBaselineService.getNetworkBaseline(serverDeploymentID)
         log.info "Requested Baseline: ${serverBaseline}"
         assert serverBaseline
```

Comment:
[not your code, but]
1. Could you please extend comment about _why_ is this needed: what would go wrong if server baseline is not requested at this point?
2. Since you overwrite `serverBaseline` variable later anyways, I suggest that you don't save anything here and just do

```groovy
assert NetworkBaselineService.getNetworkBaseline(serverDeploymentID).tap { log.info "Requested Baseline: ${it}" }
```